GET_STARTED.md

```diff
@@ -1,45 +1,89 @@
 
 # Get started!
 
-To standardize data with the pipe, you need some data! Let's set up a database filled with the MIMIC III Clinical Database Demo.
+Learn how to standardize data using the pipe!
 
-## Configuration & Setup
+## 1 Configuration
 
-First, register [on the Physionet website](https://mimic.physionet.org/gettingstarted/demo/) on the official website to get access to the demo data, it takes 30 seconds and you will get a username and a password. Access [this page](https://physionet.org/works/MIMICIIIClinicalDatabaseDemo/) to sign the agreement needed to download the data.
+But first, you need some data! We will use the MIMIC III Clinical Database Demo, for which you need to get credentials. It's quite straightforward: register [on the Physionet website](https://mimic.physionet.org/gettingstarted/demo/) to get access to the demo data, it takes 30 seconds and you will get a username and a password. Then, access [this page](https://physionet.org/works/MIMICIIIClinicalDatabaseDemo/) to sign the agreement needed to download the data (_you don't need to download the data yourself_) or click on this button:
 
-Then copy `.env.example` into `.env` and edit this last to configure it with your physionet credentials.
+[![Physionet Agreement](https://img.shields.io/badge/Physionet-Sign%20Agreement-green.svg?style=for-the-badge)](https://physionet.org/pnw/a/self-register?project=/works/MIMICIIIClinicalDatabaseDemo/index.shtml)
+
+
+Then, copy `.env.example` into `.env` and edit the file last to add your physionet credentials.
 
 ```
 cp .env.example .env
+vi .env
 source .env
 ```
 
-In the fhir-pipe repo, copy `config_demo.yml` (from the `fhirpipe` directory) into `config.yml` and put there your credentials, check that the ports are valid.
+## 2-A Docker Setup
 
-```
-cp config_demo.yml config.yml
-```
+You can use Docker to start quickly playing with demos. Alternatively, the **[2.B Manual Setup](#2-b-manual-setup)** section explains how to install the pipeline step by step.
 
-## Launch the pipe
+In the `fhirpipe` directory, copy `config_docker.yml` into `config.yml`.
 
+```
+cp fhirpipe/config_docker.yml fhirpipe/config.yml
+```
 
+Start [or install](https://docs.docker.com/install/#supported-platforms) Docker, and then build the container.
 ```
 docker-compose up --build arkhn-pipe-mimic
 ```
 
-Then, connect to the pipe container:
+Then, switch to another tab and connect to the pipeline container:
 
 ```
 docker exec -ti arkhn-pipe-mimic /bin/bash
```

Comment:
Why not use
```
docker exec -ti fhirpipe-run --project=Mimic --use-graphql-file=True
```
as usually seen when using `docker exec` ?

Reply:
You're right it would be more standard, but then I need to provide separate instructions in section 3 depending on whether the install is local or using Docker. We might and doing this anyway

Reply:
Makes sense, clarity beats convention here 👍